lib/node_modules/@stdlib/math/base/special/fmod/src/main.c

```diff
@@ -102,86 +102,83 @@ double stdlib_base_fmod( const double x, const double y ) {
 	// Determine ix = ilogb(x)
 	if ( hx < 0x00100000 ) {
 		// subnormal x
-	    if ( hx == 0 ) {
+		if ( hx == 0 ) {
 			ix = -1043;
 			for ( i = lx; i > 0; i <<= 1 ) {
 				ix -=1;
 			}
-	    } else {
+		} else {
 			ix = STDLIB_CONSTANT_FLOAT64_MIN_BASE2_EXPONENT;
 			for ( i = ( hx << 11 ); i > 0; i <<= 1 ) {
 				ix -=1;
 			}
-	    }
+		}
 	} else {
 		ix = ( hx >> 20 ) - STDLIB_CONSTANT_FLOAT64_EXPONENT_BIAS;
 	}
 
-    // determine iy = ilogb(y)
+	// determine iy = ilogb(y)
 	if ( hy < 0x00100000 ) {
 		// subnormal y
-	    if ( hy == 0 ) {
+		if ( hy == 0 ) {
 			iy = -1043;
 			for ( i = ly; i > 0; i <<= 1 ) {
 				iy -=1;
 			}
-	    } else {
+		} else {
 			iy = STDLIB_CONSTANT_FLOAT64_MIN_BASE2_EXPONENT;
 			for ( i = ( hy << 11 ); i > 0; i <<= 1) {
 				iy -=1;
 			}
-	    }
+		}
 	} else {
 		iy = ( hy >> 20 ) - STDLIB_CONSTANT_FLOAT64_EXPONENT_BIAS;
 	}
 
-    // set up {hx,lx}, {hy,ly} and align y to x
+	// set up {hx,lx}, {hy,ly} and align y to x
 	if ( ix >= STDLIB_CONSTANT_FLOAT64_MIN_BASE2_EXPONENT ) {
 		hx = 0x00100000 | ( STDLIB_CONSTANT_FLOAT64_HIGH_WORD_SIGNIFICAND_MASK & hx );
-	}
-	else {
+	} else {
 		// subnormal x, shift x to normal
-	    n = STDLIB_CONSTANT_FLOAT64_MIN_BASE2_EXPONENT - ix;
-	    if ( n <= 31 ) {
-	        hx = ( (uint32_t)hx << n ) | ( lx >> ( 32 - n ) );
-	        lx <<= n;
+		n = STDLIB_CONSTANT_FLOAT64_MIN_BASE2_EXPONENT - ix;
+		if ( n <= 31 ) {
+			hx = ( (uint32_t)hx << n ) | ( lx >> ( 32 - n ) );
+			lx <<= n;
 	    } else {
 			hx = lx << ( n - 32 );
 			lx = 0;
-	    }
+		}
 	}
 	if ( iy >= STDLIB_CONSTANT_FLOAT64_MIN_BASE2_EXPONENT ) {
 		hy = 0x00100000 | ( STDLIB_CONSTANT_FLOAT64_HIGH_WORD_SIGNIFICAND_MASK & hy );
-	}
-	else {
+	} else {
 		// subnormal y, shift y to normal
-	    n = STDLIB_CONSTANT_FLOAT64_MIN_BASE2_EXPONENT - iy;
-	    if ( n <= 31 ) {
-	        hy = ( (uint32_t)hy << n ) | ( ly >> ( 32 - n ) );
-	        ly <<= n;
-	    } else {
+		n = STDLIB_CONSTANT_FLOAT64_MIN_BASE2_EXPONENT - iy;
+		if ( n <= 31 ) {
+			hy = ( (uint32_t)hy << n ) | ( ly >> ( 32 - n ) );
+			ly <<= n;
+		} else {
 			hy = ly << ( n - 32 );
 			ly = 0;
-	    }
+		}
 	}
 	n = ix - iy;
 	while ( n-- ) {
-	    hz = hx - hy;
+		hz = hx - hy;
 		lz = lx - ly;
 		if ( lx < ly ) {
 			hz -= 1;
 		}
-	    if ( hz < 0 ) {
+		if ( hz < 0 ) {
 			hx = hx + hx + ( lx >> 31 );
 			lx += lx;
-		}
-	    else {
-	    	if ( ( hz | lz ) == 0 )
+		} else {
+			if ( ( hz | lz ) == 0 )
```

Comment:
@gunjjoshi I am not following. Why is L176-177 any different than L194-197? Why would making the change to wrap in `{}` cause test failures?

Reply:
I realized the mistake. What I was doing was:

```c
if ( ( hz | lz ) == 0 ) {
	// return sign(x)*0
}
return ZERO[ (uint32_t)sx >> 31 ];
```
( which is a completely different thing than what we expect here )

instead of:

```c
if ( ( hz | lz ) == 0 ) {
        // return sign(x)*0
	return ZERO[ (uint32_t)sx >> 31 ];
}
```